crates/sncast/tests/e2e/tx_status.rs

```diff
@@ -17,7 +17,7 @@ async fn test_incorrect_transaction_hash() {
         output,
         indoc! {r"
         command: tx-status
-        error: Failed to get transaction status: TransactionHashNotFound
+        error: Failed to get transaction status: Transaction with provided hash was not found (does not exist)
```

Comment:
Where does this change exactly come from?

Reply:
I added [here](https://github.com/foundry-rs/starknet-foundry/blob/2644d6f00dd449f1f68d1d8c4f5fb8c63639f095/crates/sncast/src/starknet_commands/tx_status.rs#L23) conversion to StarknetCommandError. Which [has](https://github.com/foundry-rs/starknet-foundry/blob/7667a6ad819ebccd963810914a6ff9f54d5479be/crates/sncast/src/response/errors.rs#L66) custom display